providers/apis/dydx/multi_market_map_fetcher_test.go

```diff
@@ -96,6 +96,7 @@ func TestDYDXMultiMarketMapFetcher(t *testing.T) {
 									CurrencyPair:     slinkytypes.NewCurrencyPair("BTC", "USD"),
 									Decimals:         8,
 									MinProviderCount: 1,
+									Enabled:          true,
```

Comment:
Does the test case below also have to have it be enabled?

Reply:
yeah, it should I guess - we are only checking that a field exists under that key.

not any of the fields themselves like `Enabled`